CSharp/grades/Student.cs

```diff
@@ -0,0 +1,22 @@
+using System.Linq;
+namespace grades;
+
+public class Student
+{
+    public int StudentNumber {get; set;}
+    public string FirstName {get; set;}
+    public string LastName {get; set;}
+    public List<Grade> Grades {get; set;}
+
+    public Student(int studentNumber, string firstName, string lastName) {
```

Comment:
when you have properties with default getters and setters, you don't need to have constructors.

Reply:
Constructor is needed to initialize the List I think.